src/components/CryptoTraq.js

```diff
@@ -24,16 +30,26 @@ const CryptoTraq = (props) => {
           <SelectionForm
               formLabel={"Select Cryptocurency:"}
               selectionFunc={props.handleCryptoChange}
-              selectForm={<CryptoSelectForm list={props.cryptoList}/>}
+              selectForm={<CryptoSelectForm list={props.cryptoList} delay={styles.delay1} anim={styles.select__glow}/>}
```

Comment:
Here I have added the delay and animation styles for the `select` element.  They are defined below with the rest of the styles.